File: Beecrowd/Beginner/1101 - 1190/1187 - Top Area/C/1187_TopArea.c
#include <stdio.h>

int main() {
    char operation;
    double matrix[12][12], sum = 0;

    scanf("%c", &operation);

    for (int i = 0; i < 12; i++) {
        for (int j = 0; j < 12; j ++) {
            scanf("%lf", &matrix[i][j]);

            if (i < 5 && j >= i + 1 && j <= 10 - i) {
                sum += matrix[i][j];
            }
        }
    }

    if (operation == 'S') printf("%.1lf\n", sum);
    else printf("%.1lf\n", sum / 30);

    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1187 - Top Area/Python/1187_TopArea.py
sum = 0

operation = input()

for i in range(12):
    for j in range(12):
        number = float(input())
        if i < 5 and j >= i + 1 and j <= 10 - i:
            sum += number
            
if operation == 'S':
    print(f"{sum:.1f}")
else:
    print(f"{sum / 30:.1f}")
